### PR TITLE
CC 1.0.0 - Updating quick start and supported features [4.0.0]

### PR DESCRIPTION
**Pull request by pubudu538**
## Purpose

- Updated the quick start guide
- Updated the logos of docker and K8s
- Updated the supported features

Outputs:

Screenshot 2021-11-13 at 12 45 11

screenbud-aff40af0-3968-4aa1-bb46-c9d7551df608

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker-with-apim.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Make sure to install and set up [Docker](https://www.docker.com).
 
 ### Step 1 - Download and setup Choreo Connect distribution zip
 
-Latest Choreo Connect distribution can be downloaded from the [page](https://wso2.com/api-management/api-microgateway/). Extract the Choreo Connect distribution zip. Extracted folder will be called as `CC_HOME` hereafter.
+Latest Choreo Connect distribution can be downloaded from the [page](https://github.com/wso2/product-microgateway/releases). Extract the Choreo Connect distribution zip. Extracted folder will be called as `CHOREO-CONNECT_HOME` hereafter.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker-with-apim.md:11]:
```suggestion
Latest Choreo Connect distribution can be downloaded from the [GitHub repository](https://github.com/wso2/product-microgateway/releases). Extract the Choreo Connect distribution .zip file. The wxtracted folder will be called as `CHOREO-CONNECT_HOME` hereafter.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ to setup the APICTL in your development environment.
 
 #### Step 1.2 - Download and Extract Choreo Connect distribution zip
 
-Latest Choreo Connect distribution can be downloaded from the [page](https://wso2.com/api-management/api-microgateway/).
+Latest Choreo Connect distribution can be downloaded from the [page](https://github.com/wso2/product-microgateway/releases).
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md:19]:
```suggestion
Latest Choreo Connect distribution can be downloaded from the [GitHub repository](https://github.com/wso2/product-microgateway/releases).
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ to setup the APICTL in your development environment.
...
 Extract the Choreo Connect distribution zip. Extracted folder will be called as `CHOREO-CONNECT_HOME` hereafter.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md:20]: Make same change suggested above here.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes-with-apim.md [hunk 4]
```diff
@@ -20,9 +20,9 @@ Let's get started...
 
 ## Step 1 - Setup Choreo Connect and K8s API Operator in Kubernetes
 
-1.  Download the Choreo Connect v0.9.0 from
-    [GitHub release page's](https://github.com/wso2/product-microgateway/releases/tag/v0.9.0) assets section and extract it
-    to a folder of your choice. Let's refer to this folder as `<CHOREO-CONNECT_HOME>`.
+1.  Download and Extract Choreo Connect distribution zip
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes-with-apim.md:23]:
```suggestion
1.  Download and extract Choreo Connect distribution .zip file.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes-with-apim.md [hunk 5]
```diff
@@ -20,9 +20,9 @@ Let's get started...
...
+
+    Latest Choreo Connect distribution can be downloaded from the [page](https://github.com/wso2/product-microgateway/releases). Extract the Choreo Connect distribution zip. Extracted folder will be called as `CHOREO-CONNECT_HOME` hereafter.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes-with-apim.md:25]:
```suggestion
    The latest Choreo Connect distribution can be downloaded from the [GitHub repository](https://github.com/wso2/product-microgateway/releases). Extract the Choreo Connect distribution .zip file. the extracted folder will be referred to as `CHOREO-CONNECT_HOME` hereafter.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes.md [hunk 6]
```diff
@@ -20,9 +20,9 @@ Let's get started...
 
 ## Step 1 - Setup Choreo Connect and K8s API Operator in Kubernetes
 
-1.  Download the Choreo Connect v0.9.0 from
-    [GitHub release page's](https://github.com/wso2/product-microgateway/releases/tag/v0.9.0) assets and extract them
-    to a folder of your choice. We will refer to this folder as the `CHOREO-CONNECT_HOME`.
+1.  Download and Extract Choreo Connect distribution zip
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes.md:23]: Please apply the same fix here.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md [hunk 7]
```diff
@@ -1,46 +1,76 @@
...
-- Integrate with Consul Service Mesh.
-- Support basic auth protected basic end services
-- Support publishing APIs to multiple gateway environments
-
-The following key features are not supported by the Choreo Connect as of now they will be added in the future releases.
-
-- Streaming APIs like SSE, web sub and etc.
-- Create streaming APIs from Async API specifications.
-- Analytics for the web socket APIs.
+Choreo Connect's most recent version v1.0.0 does not support all of the API Manager's functionalities. In contrast to default API gateway, Choreo Connect (the API Microgateway) only offers a fraction of the API Manager product's functionality. The following is a list of Choreo Connect's important features.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md:3]:
```suggestion
Choreo Connect's most recent version (v1.0.0) does not support all of the API Manager's functionalities. In contrast to the default API Gateway, Choreo Connect (the API Microgateway) only offers a fraction of the API Manager product's functionality. The following is a list of Choreo Connect's important features.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md [hunk 8]
```diff
@@ -1,46 +1,76 @@
...
+- Streaming APIs like SSE, web sub and etc
+- Create streaming APIs from Async API specifications
+- Analytics for the web socket APIs
 - GraphQL APIs
-- API Products.  
-- Mutual SSL authentication for APIs.
-- Application level securities like basic auth and API keys.
-- Message transformation and mediation.
-- Bandwidth based throttling.
+- Grpc APIs
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md:70]:
```suggestion
- gRPC APIs
```

**Comment by samgnaniah** [comment]
Please apply the same fixes to the PR going into the master branch.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker-with-apim.md (hunk 1 above)
Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker-with-apim.md]: Fixed with https://github.com/wso2/docs-apim/pull/5012/commits/877891c33aed33fd26768308653306d191562895

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md (hunk 2 above)
Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md]: Fixed with https://github.com/wso2/docs-apim/pull/5012/commits/877891c33aed33fd26768308653306d191562895

**Comment by pubudu538** [comment]
Changes are done with https://github.com/wso2/docs-apim/pull/5012/commits/877891c33aed33fd26768308653306d191562895

**Comment by pubudu538** [comment]
I have done the requested changes in the master branch PR as well.

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**